main/ertp/guide/purses-and-payments.md

```diff
@@ -1,109 +1,107 @@
 # Purses and Payments
- 
-Digital assets can be any of:
-- Currency-like, such as our imaginary Quatloos currency.
-- Goods-like digital assets, such as magic weapons for use in a game or theater tickets.
-- Other kinds of rights, such as the right to participate in a particular contract.
 
-In ERTP, digital assets exist in either a `purse` or a `payment`:
+There are different kinds of digital assets:
+- Currency-like, such as our imaginary Quatloos.
+- Goods-like, such as theater tickets or magic weapons for use in a game.
+- Abstract rights, such as authorization to participate in a particular contract.
+
+In ERTP, digital assets always exist in either a `purse` or a `payment`:
 - **`purse`**: Holds
   an amount of same-branded digital assets until part or
   all of them are withdrawn into a `payment`. A new `purse` is created
-  by an `issuer` and can only hold assets of that `issuer`'s `brand`. 
+  by an `issuer` and can only hold assets of that `issuer`'s `brand`.
 - **`payment`**:
-  Holds a quantity of same-branded digital assets to transfer to another party. 
-  A new `payment` is created either with new assets by a `mint` or by 
-  withdrawing assets from a `purse`. It can only hold assets of the same `brand` as
-  that `mint` or `purse`.
+  Holds a quantity of same-branded digital assets to transfer to another party.
+  A `payment` is created containing either new assets from a `mint` or existing assets
+  withdrawn from a `purse` or transferred from one or more other consumed `payment`s.
+  It can only hold assets of the same `brand` as its source(s).
 
-For any `brand`, any number of `purses` or `payments` can hold assets
-of that `brand`. Neither a `purse` nor a `payment` can ever change their
+Any number of `purses` or `payments` can hold assets
+of any particular `brand`. Neither a `purse` nor a `payment` can ever change their
 associated `brand`.
 
-Each `purse` and `payment` object contains a specific single amount of digital assets, 
-which may be none at all (`empty` in AmountMath terms). In the same way 
-you might have separate bank accounts for different purposes, 
-you can have different purses for the same brand of digital asset. 
-One purse might hold 2 Quatloos and another purse might hold 3 Quatloos. 
+Each `purse` and `payment` object contains a specific single amount of digital assets,
```

Comment:
```suggestion
Each `purse` and `payment` object contains a specific amount of digital assets,
```
I think "single" is confusing here. Is a set of assets a singular something or a plural something?